### ISSUE TITLE
'command_validate_remote_bot' - failing

### ISSUE THREAD

**Issue by reddy75** [reddy75/wrecon#6]
```
python: stdout/stderr (wrecon): Traceback (most recent call last):
python: stdout/stderr (wrecon):   File "/home/reddy/.weechat/python/autoload/wrecon.py", line 2071, in parse_buffer
python: stdout/stderr (wrecon):     SCRIPT_BUFFER_CALL[v_cmd](data, buffer, xtags, prefix, v_arguments)
python: stdout/stderr (wrecon):   File "/home/reddy/.weechat/python/autoload/wrecon.py", line 1119, in receive_advertise_additionally
python: stdout/stderr (wrecon):     command_validate_remote_bot(xdata, xbuffer, xcmd, xtags, xprefix, xargs)
python: stdout/stderr (wrecon):   File "/home/reddy/.weechat/python/autoload/wrecon.py", line 1890, in command_validate_remote_bot
python: stdout/stderr (wrecon):     if args[1] == wrecon_bot_id:
python: stdout/stderr (wrecon): IndexError: list index out of range
python: chyba ve funkci "parse_buffer"
```

**Comment by reddy75** [comment]
Bug fix in progress

**Comment by reddy75** [comment]
```
10:28:55 @xxxxxxxx | WRECON-CMD>E-ADA aaaaaaaaaaaaaaaa bbbbbbbbbbbbbbbb 005-AJl [v1.16 20200222193206CET]                                                                                                         │
10:28:56 @xxxxxxxx | WRECON-CMD>ADA-R bbbbbbbbbbbbbbbb aaaaaaaaaaaaaaaa 005-AJl [v1.15 20200222184352CET]                                                                                               │
10:28:56  [wrecon] | REMOTE BOT REGISTERED : aaaaaaaaaaaaaaaa ([v1.15 20200222184352CET])  
```
command happened between versions 1.15 and 1.16

**Comment by reddy75** [comment]
Issue has been already fixed in version 1.16
